helm/flux-app/Chart.yaml

```diff
@@ -1,10 +1,19 @@
-apiVersion: v1
-appVersion: 0.19.0
-description: A Helm chart for flux
+apiVersion: v2
+appVersion: 0.21.0
+annotations:
+  application.giantswarm.io/team: "honeybadger"
+description: |
+  Flux is a collection of tools for keeping Kubernetes clusters in sync
+  with sources of configuration (like Git repositories), and automating
+  updates to configuration when there is new code to deploy.
 engine: gotpl
 home: https://github.com/giantswarm/flux-app
 icon: https://s.giantswarm.io/app-icons/flux/1/dark.svg
+keywords:
+- gitops
+- flux
 name: flux-app
 sources:
 - https://github.com/fluxcd/flux2
+type: application
```

Comment:
Nice to include this too. 👍

Reply:
Let's make sure we add our custom metadata as well. Can this chart be deployed multiple times per namespace / cluster?
https://intranet.giantswarm.io/docs/product/architecture-specs-adrs/specs/managed-apps/2020-05-05-app-versions-representation/#supported-chartyaml-file-format